oss-internship-2020/pffft/CMakeLists.txt

```diff
@@ -0,0 +1,91 @@
+cmake_minimum_required(VERSION 3.10)
+
+project(pffft CXX C)
+
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED True)
+
+add_library(pffft STATIC 
+    pffft.c 
+    pffft.h 
+    fftpack.c 
+    fftpack.h
+)
+
+add_executable(pffft_main
+    test_pffft.c
+)
+
+target_link_libraries(pffft_main PRIVATE 
+    pffft
+)
+
+set(MATH_LIBS "")
+include(CheckLibraryExists)
+check_library_exists(m sin "" LIBM)
+if(LIBM)
+    list(APPEND MATH_LIBS "m")
+endif()
+
+target_link_libraries(pffft PUBLIC ${MATH_LIBS})
+
+
+# Adding dependencies
+set(SAPI_ROOT "/usr/local/google/home/inach/sandboxed-api" CACHE PATH "Path to the Sandboxed API source tree")
```

Comment:
Is it possible to set it to ../sandboxed-api ? I mean... we probably should not introduce local paths to those files, even as defaults. I suggest removing the default value.

Still, maybe simply ../sandboxed-api or .. or ../.. will work as a default value well?